simulations/vip-160/vip-160/simulation.ts

```diff
@@ -0,0 +1,166 @@
+    let shortfall: Contract;
+
+    before(async () => {
+      shortfall = await ethers.getContractAt(SHORTFALL_ABI, SHORTFALL);
+    });
+
+    it("should accept ownership over the Shortfall", async () => {
+      expect(await shortfall.owner()).to.equal(NORMAL_TIMELOCK);
+    });
+
+    it("should have the PoolRegistry address set", async () => {
+      expect(await shortfall.poolRegistry()).to.equal(POOL_REGISTRY);
+    });
+
+    it("should have the RiskFund address set", async () => {
+      expect(await shortfall.riskFund()).to.equal(RISK_FUND);
+    });
+
+    it("should have minimumPoolBadDebt equal to $1000", async () => {
+      expect(await shortfall.minimumPoolBadDebt()).to.equal(parseUnits("1000", 18));
+    });
+
+    it("should have incentive equal to 10% (1000bps)", async () => {
+      expect(await shortfall.incentiveBps()).to.equal(1000);
+    });
+
+    it("should have waitForFirstBidder equal to 100 blocks", async () => {
+      expect(await shortfall.waitForFirstBidder()).to.equal(100);
+    });
+
+    it("should have nextBidderBlockLimit equal to 100 blocks", async () => {
+      expect(await shortfall.nextBidderBlockLimit()).to.equal(100);
+    });
+
+    it("should have convertibleBaseAsset equal to USDT", async () => {
+      expect(await shortfall.convertibleBaseAsset()).to.equal(USDT);
+    });
+  });
+
+  describe("ProtocolShareReserve configuration", () => {
+    let protocolShareReserve: Contract;
+
+    before(async () => {
+      protocolShareReserve = await ethers.getContractAt(PROTOCOL_SHARE_RESERVE_ABI, PROTOCOL_SHARE_RESERVE);
+    });
+
+    it("should accept ownership over the ProtocolShareReserve", async () => {
+      expect(await protocolShareReserve.owner()).to.equal(NORMAL_TIMELOCK);
+    });
+
+    it("should have the PoolRegistry address set", async () => {
+      expect(await protocolShareReserve.poolRegistry()).to.equal(POOL_REGISTRY);
+    });
+
+    it("should have the RiskFund address set", async () => {
+      expect(await protocolShareReserve.riskFund()).to.equal(RISK_FUND);
+    });
+
+    it("should send the remaining income to Venus Treasury", async () => {
+      expect(await protocolShareReserve.protocolIncome()).to.equal(VENUS_TREASURY);
+    });
+  });
+
+  describe("Markets configuration", () => {
+    before(async () => {
+      poolRegistry = await ethers.getContractAt(POOL_REGISTRY_ABI, POOL_REGISTRY);
+    });
+
+    it("sets ProtocolShareReserve address for all markets", async () => {
+      const markets = await getAllMarkets(poolRegistry);
+      for (const market of markets) {
+        expect(await market.protocolShareReserve()).to.equal(PROTOCOL_SHARE_RESERVE);
+      }
+    });
+
+    it("sets Shortfall contract address for all markets", async () => {
+      const markets = await getAllMarkets(poolRegistry);
+      for (const market of markets) {
+        expect(await market.shortfall()).to.equal(SHORTFALL);
+      }
+    });
+  });
```

Comment:
Would it make sense to test the happy path, invoking the `reduceReserves` function on the VToken contracts, and them the `releaseFunds`, to finally check the amounts in the Treasury and the RiskFund contract? I know this is part of the test suite, so, maybe it's not needed. Wdyt?

Reply:
[c038a61](https://github.com/VenusProtocol/vips/pull/59/commits/c038a61b682c6cc16b1a45aa90f76eb19f2bf6a4)